h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostPredictImplComparisonTest.java

```diff
@@ -97,14 +93,17 @@ public void testPredictionsAreSame() throws IOException {
             System.setProperty("sys.ai.h2o.xgboost.predict.native.enable", "false");
             Frame predsJava = Scope.track(model.score(testFrame));
 
-            assertFrameEquals(predsNative, predsJava, getDelta());
+            assertFrameEquals(predsNative, predsJava, getDelta(), true);
         } finally {
             Scope.exit();
         }
     }
     
     private double getDelta() {
-        if ("gbtree".equals(booster)) {
+        if (XGBoost.hasGPU(H2O.CLOUD.members()[0], 0)) {
```

Comment:
we should probably make the decision based on if GPU was actually used to train the algo not just if it is available? it might be the same thing in the current state of the test - but if things changed it wouldn't be obvious how is it actually being tested

Reply:
fixed